.travis.yml

```diff
@@ -21,6 +24,9 @@ before_script:
 script:
   - npm test
   # Only run funcs for LTS releases
+  - if [ "$TRAVIS_NODE_VERSION" = "11" ]; then npm run functionaltest; fi
+  - if [ "$TRAVIS_NODE_VERSION" = "10" ]; then npm run functionaltest; fi
+  - if [ "$TRAVIS_NODE_VERSION" = "8.0.0" ]; then npm run functionaltest; fi
```

Comment:
I wonder if we should have a comment explaining why we add 8.0.0 in particular in addition to 8

Reply:
Added comment